test/membrane_mp4/integration_test.exs

```diff
@@ -4,12 +4,14 @@ defmodule Membrane.MP4.IntegrationTest do
   alias Membrane.MP4.Container
   alias Membrane.Testing
 
+  # Fixtures used in muxer tests below were generated with `samples_per_chunk` option set to `10`.
+
   test "video" do
     children = [
       file: %Membrane.File.Source{location: "test/fixtures/in_video.h264"},
       parser: %Membrane.H264.FFmpeg.Parser{framerate: {30, 1}, attach_nalus?: true},
       payloader: Membrane.MP4.Payloader.H264,
-      cmaf: Membrane.MP4.CMAF.Muxer,
```

Comment:
Has that test disappeared?

Reply:
It has been moved to `Membrane.MP4.Muxer.CMAF.IntegrationTest`.